src/database/models/user.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+import { Model } from 'sequelize';
+module.exports = (sequelize, DataTypes) => {
+  class User extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    // static associate(models) {
+    // define association here
+    // }
+  }
```

Comment:
You can use another syntax provided by NestJS for creating model, check this: https://docs.nestjs.com/techniques/database#models
It will be better when the project will be written in TypeScript when it's possible. Thougths?